steps/representative/representative-details/RepresentativeDetails.js

```diff
@@ -18,9 +20,16 @@ class RepresentativeDetails extends Question {
         return paths.representative.representativeDetails;
     }
 
+    get CYAName() {
+
+        const firstName = this.fields.nameOrganisation.firstName.value || '';
+        const lastName = this.fields.nameOrganisation.lastName.value || '';
+        return firstName === '' && lastName === '' ? userAnswer.NOT_PROVIDED : `${firstName} ${lastName}`.trim();
```

Comment:
`${firstName} ${lastName}.trim();` removes only the trailing white space from `firstName` and the `leading` white space from `lastName`. Everything in between will not be removed.

Reply:
Actually, this may not be the case due to the `Joi.string().trim()` on both `firstName ` and `lastName`.

Reply:
@paulgain This is what I want. I only want to remove whitespace before and after the entire string. This is because of this scenario:
User only enters lastName (not firstName) which would make the string look like this: 
` lastName` (whistespace before the lastName). 
So trim() is there to remove that.